### DISCUSSION TITLE
How to set width and height for nodes in rectangle shape?

### DISCUSSION THREAD

**Discussion by mbao002-baomic** [yWorks/yfiles-jupyter-graphs#36] [Q&A]
Hi there, 
Which property can be used to set size (width and height) of rectangle nodes? Custom_size_factor just makes them small or large squares. I'm looking for flexibility to fix labels inside a normal rectangle shape. Also any way to make labels complete transparent without background colour?

Where can I find more detailed documentation on this widget, in particular how to set properties on node and edge. List of properties or attributes available inside related dict parameters. Thanks
Michael

**Comment by yGuy** [comment]
Thanks for the feedback. These are actually "missing features" and you might want to file an issue for it. Both requests are absolutely reasonable and we would like to fix them, eventually. 
That said, for completeness sake, the original intent for building the widget was not for creating a tool that is primarily meant to create pixel-perfect graphs for printing, but for making graph visualization easily available in python notebooks. As such, the goal is not to provide control over each and every aspect of the visualization and some things are likely to stay hard-coded for the sake of simplicity of the API and UI. 
But you can always export your graph to yEd-live and then manually modify various aspects of the visualization to create export-quality visualizations. Just hit the "open in yEd-live" button and start modifying the visualization.
